sdk/core/azure-core/CMakeLists.txt

```diff
@@ -42,14 +42,18 @@ if(BUILD_TRANSPORT_CURL)
     src/http/curl/curl_connection_pool_private.hpp
     src/http/curl/curl_connection_private.hpp
     src/http/curl/curl_session_private.hpp
-  )
+   )
   SET(CURL_TRANSPORT_ADAPTER_INC
     inc/azure/core/http/curl_transport.hpp
   )
 endif()
 if(BUILD_TRANSPORT_WINHTTP)
-  SET(WIN_TRANSPORT_ADAPTER_SRC src/http/winhttp/win_http_transport.cpp)
-  SET(WIN_TRANSPORT_ADAPTER_INC inc/azure/core/http/win_http_transport.hpp)
+  SET(WIN_TRANSPORT_ADAPTER_SRC 
+    src/http/winhttp/win_http_transport.cpp
+  )
+ SET(WIN_TRANSPORT_ADAPTER_INC 
+    inc/azure/core/http/win_http_transport.hpp
+ )
```

Comment:
Actually, we want to keep the old one because the websockets PR will be merging with these changes. Since the whole purpose of this PR is to reduce merges with websockets, I want to make those merges as easy as possible.

Reply:
But then we get more unnecessary changes here and now, I think the right way to fix is to propagate that change to that branch that you are referring to.

Reply:
The entire reason this pull request exists is to reduce the number of merge conflicts when merging main into the feature/websockets branch. Making your suggested change *increases* the number of merge conflicts, which seems to be counter to the entire effort.

Reply:
But we're introducing unnecessary changes to `main`, formatting lines. Shouldn't the other branch get in alignment with `main` before merging, and not the other way?

Reply:
When websockets merges into main, it will be introducing new files to both the WIN_TRANSPORT_ADAPTER_SRC and WIN_TRANSPORT_ADAPTER_INC. That will require that we reformat the lines.

There is no way of changing websockets to conform to what main currently has because the formatting for "set" with one value is different from the formatting of "set" with two values.

The entire purpose of htis pull request is to reduce the amount of work needed to merge main with websockets. Changing this line will *increase* the amount of work to merge main with websockets. And the instant that websockets merges with main, the formatting inconsistency will be resolved.

Reply:
Ok, that explains it.